mftf/2.3/configuration.md

```diff
@@ -178,7 +186,7 @@ TESTS_BP=~/magento2ce/dev/tests/acceptance
 
 #### FW_BP
 
-The path to where MFTF exists and from which it is symlinked.
+The path to where the MFTF exists and from which it linked using symbolic links.
```

Comment:
The path to MFTF. 

(I don't think you need to mention this being the origin of a symlink again.)

Reply:
:ok: